CaptchaBuilder.php

```diff
@@ -128,19 +133,15 @@ public function setInterpolation($interpolate = true)
      */
     public $tempDir = 'temp/';
 
-    public function __construct($phrase = null, PhraseBuilderInterface $builder = null)
```

Comment:
I agree that phase can be generated in `build()` if `null`, but I think we shouldn't break bc and let the possibility to set the phrase in the constructor